src/CodeViewerDemo/CodeExamples.h

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <QApplication>
+
+const QString cpp_example = R"(#pragma twice
```

Comment:
If you define these variables in a header you have to declare them `inline`. Otherwise you will get linker error when including them from different cpp-files.

Reply:
Yes, thanks!